src/components/general/Search/search.module.scss

```diff
@@ -0,0 +1,50 @@
+$border_color: #BACAD1;
+$bg_color: #FFFFFF;
+$font_color: #2E2E22;
+$placeholder_font_color: #354E57;
+
+.search__input {
+    width: 350px;
+    border: none;
+    border-bottom: 1px solid $placeholder_font_color ;
+    background-color: transparent;
+    padding: 10px;
+}
+
+input::placeholder {
+    color: $placeholder_font_color;
+    font-size: 16px;
+}
+
+.results__container {
+    position: absolute;
+    z-index: 1;
+}
+
+.results__wrapper {
+    padding: 15px;
+    max-width: 350px;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    border: 1px solid $border_color;
+    background-color: $bg_color;
+}
+
+.results__image {
+    width: 100px;
+    height: 100px;
+    border-right: 1px solid $border_color;
+    padding-right: 15px;
+}
+
+.results__title {
+    font-weight: 100;
+    font-size: 14px;
+    max-width: 180px;
+    color: $font_color;
+    white-space: nowrap;
+  overflow: hidden;
```

Comment:
identation should be equal, not sure if it's github review, but if it's not, please fix this